aspnetcore/fundamentals/minimal-apis/includes/parameter-binding8.md

```diff
@@ -100,7 +100,7 @@ Parameters declared in route handlers are treated as required:
 
 To make `pageNumber` optional, define the type as optional or provide a default value:
 
-[!code-csharp[](~/fundamentals/minimal-apis/7.0-samples/WebMinAPIs/Program.cs?name=snippet_op2)]
+:::code language="csharp" source="~/fundamentals/minimal-apis/7.0-samples/WebMinAPIs/Program.cs" id="snippet_op2" highlight="4,6-8":::
```

Comment:
Added highlights for the optional or default value:

![image](https://github.com/dotnet/AspNetCore.Docs/assets/28659384/e0fce5f5-09a9-4e7d-92e2-0e264a2796b7)